### PR TITLE
small cleanups and add documentation for file fetching methods

### PR DESCRIPTION
**Pull request by cameroncuster**
- removed files that I don't think we actually care about
- added docs for the file fetching methods

### CODE REVIEW

File: src/setupTests.ts
Comment (AsirAAlam) [inline src/setupTests.ts:1]: we might need this if we want to set up jest tests

**Review by tylerm390 — APPROVED** [review]
lgtm

File: src/setupTests.ts
Comment (cameroncuster) [reply src/setupTests.ts:1]: no I already added jest tests, this isn't necessary

File: src/react-app-env.d.ts
Comment (AsirAAlam) [inline src/react-app-env.d.ts:1]: we might need this if we want to declare modules. for example, for importing non-ts files

Comment (cameroncuster) [reply src/react-app-env.d.ts:1]: I thought `*.d` was always like built files to be honest

**Pull request merged**